extension/background.js

```diff
@@ -0,0 +1,54 @@
+"use strict";
+
+let lastTabID;
+
+async function showPage(info) {
+  // If we previously opened a tab, reuse it.
+  if (lastTabID !== undefined) {
+    let tab = await browser.tabs.get(lastTabID).catch();
```

Comment:
`.catch()` is equivalent to `.then(undefined, undefined)`, which doesn't handle the error. To unconditionally ignore the error, pass a no-op callback, e.g. an empty arrow function.

Instead of saving the tabId, you could also query for the expected base URL by the way.